packages/typespec-ts/test/azureModularIntegration/generated/client/namespace/src/index.d.ts

```diff
@@ -0,0 +1,41 @@
+import { ClientOptions } from '@azure-rest/core-client';
+import { OperationOptions } from '@azure-rest/core-client';
+import { Pipeline } from '@azure/core-rest-pipeline';
+
+export declare class ClientNamespaceFirstClient {
```

Comment:
Could you add the js decorator part in spector side? https://github.com/Azure/typespec-azure/pull/1929/files#r1953860516